### ISSUE TITLE
Missing semi colon at the end of rastercoords.js

### ISSUE THREAD

**Issue by EduardoFromTheFuture** [commenthol/leaflet-rastercoords#5]
I've got a project that uses bower to install "leaflet-rastercoords": "^1.0.2".

The file being used by my project, <script src="/bower_components/leaflet-rastercoords/rastercoords.js"></script>, must be the last included file in my index.html, otherwise the following error is risen:
"Uncaught TypeError: (intermediate value)(intermediate value)(...) is not a function"
The fix is to simply add a semicolon ";" at the end of the file.

In short, rastercoords.js is missing a ; at the end, and if there are other scripts loaded after it, an error appears on the browser console. The workaround is to make rastercoords the last script to be loaded.

**Comment by commenthol** [comment]
Hi, The issue was fixed in version 1.0.3

**Comment by EduardoFromTheFuture** [comment]
Good job! **Thanks** for the quickness!!
